dbt/adapters/databricks/connections.py

```diff
@@ -475,6 +473,8 @@ def add_query(
         auto_begin: bool = True,
         bindings: Optional[Any] = None,
         abridge_sql_log: bool = False,
+        retryable_exceptions: tuple[type[Exception], ...] = tuple(),
```

Comment:
This is new interface in latest upstream.  Hopefully this doesn't break anything, otherwise we (and other adapters) will be in for a world of hurt.